src/test/groovy/org/prebid/server/functional/testcontainers/PbsConfig.groovy

```diff
@@ -71,10 +71,11 @@ LIMIT 1
     }
 
     static Map<String, String> getBidderConfig(String rootUri = networkServiceContainer.rootUri) {
-        ["adapters.generic.enabled"                    : "true",
-         "adapters.generic.endpoint"                   : "$rootUri/auction".toString(),
-         "adapters.generic.usersync.cookie-family-name": "generic",
-         "adapters.generic.ortb-version"               : "2.6"]
+        ["adapters.generic.enabled"                                 : "true",
+         "adapters.generic.endpoint"                                : "$rootUri/auction".toString(),
+         "adapters.generic.aliases.cwire.meta-info.site-media-types": "",
```

Comment:
1) Please add a comment
`// due to a config validation we'll need to circumvent all future aliases this way`
2) Please move to the bottom as there will be more aliases of Generic bidder.